avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDeConfig.java

```diff
@@ -61,6 +63,19 @@
   public static final String SCHEMA_REGISTRY_USER_INFO_DOC =
       "Specify the user info for Basic Auth in the form of {username}:{password}";
 
+  public static final String KEY_SUBJECT_NAME_STRATEGY = "key.subject.name.strategy";
+  public static final String KEY_SUBJECT_NAME_STRATEGY_DEFAULT =
+      TopicNameStrategy.class.getName();
+  public static final String KEY_SUBJECT_NAME_STRATEGY_DOC =
+      "Determines how to construct the subject name under which the key schema is registered "
+      + "with the schema registry";
```

Comment:
Minor nit: perhaps the key and value doc strings could say that by default the older topic naming behavior/strategy is used. It's not essential, but if it can be said clearly and succinctly it might help people understand that the behavior won't change if they use the default.

Reply:
Added clarification of default behaviour in bf574fea86d3163962ebdb18d72de3db3f801a4b.